CODE_OF_CONDUCT.md

```diff
@@ -1,17 +1,8 @@
-# Code of Conduct
+# OpenJS Foundation Code of Conduct
 
-The OpenJS Foundation and its member projects use [Contributor Covenant v2.0](https://contributor-covenant.org/version/2/0/code_of_conduct) as their code of conduct. The full text is included below in English, and [translations](https://www.contributor-covenant.org/translations) are available on the Contributor Covenant website.
+## Report an incident
```

Comment:
IMO we should mention at the top where people can read our Code of Conduct.

Reply:
This document _is_ the code of conduct itself. Can you clarify your suggestion?

Reply:
The Code of Conduct itself is not mentioned here, just the guidelines of reporting incidents. The actual code of conduct that is in another document should actually be included or at least referenced here.

Reply:
Right, but it pretty much only explains the reporting pieces of our CoC, it doesn't mention the bylaws and the actual ettiquete and rules that the community should follow.

Hence, mentioning the remaining documents here, would make sense. We were using the Contributor Covenant, and we mentioned it here. Now we are moving towards an in-house baked guideline, but it's not mentioned anywhere within this document.

Reply:
We're still using the Contributor Covenant. CODE_OF_CONDUCT.md _is_ the code of conduct and _is_ Contributor Covenant. From the perspective of someone reporting an incident, little has changed.

Reply:
I don't see any mention of the Contributor Covenant anywhere. I think its fundamental that there are mentions of the actual code of conduct.

How will people know what are the rules they should abide by? I don't see a mention of this anywhere.

Reply:
The content of `CODE_OF_CONDUCT.md` is (and always has been) the Contributor Covenant. I've just removed the reference to if at the top to avoid getting people confused between the version that we are running and the version on the Contributor Covenant website, as we've agreed to partially modify it and it's now only 99.5% equivalent.

Reply:
That might be the source of my confusion here. Let's chat over the meeting today.